src/components/LoginForm/LoginForm.jsx

```diff
@@ -1,16 +1,94 @@
 import React, { Component } from 'react';
 import { axiosApiInstance as API } from '../../utils/axiosConfig';
 import LocalStorageService from '../../utils/localStorageService';
-import { Button, TextField, Dialog, DialogActions, DialogContent, DialogTitle, CircularProgress } from '@material-ui/core';
+import {
+  Avatar,
+  Button,
+  CssBaseline,
+  TextField,
+  FormControlLabel,
+  Checkbox,
+  Link,
+  Paper,
+  Box,
+  Grid,
+  Typography,
+  CircularProgress,
+  Backdrop,
+} from '@material-ui/core';
+import Alert from '@material-ui/lab/Alert';
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = (theme) => ({
```

Comment:
@thomasdahlberg I wanted to have a separate file for these styles, but I can't find the solution yet while it's very simple with Hooks. I could technically rewrite the entire component with Hooks, but let me know if you know how to do it with the class-based component.

Reply:
I'm not sure how you would do it with a class-based component either.